plugins/serializers/ubazzar/ubazzar_json.go

```diff
@@ -0,0 +1,135 @@
+package ubazzar
+
+import (
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/gofrs/uuid"
+
+	"github.com/influxdata/telegraf"
+)
+
+type serializer struct {
+	TimestampUnits time.Duration
+}
+
+type event struct {
+	EventID           string            `json:"event_id"`
+	ServiceCustomerID string            `json:"service_customer_id"`
+	Service           string            `json:"service"`
+	UnitOfMeasure     string            `json:"unit_of_measure"`
+	Quantity          float64           `json:"quantity"`
+	StartTime         string            `json:"start_time"`
+	EndTime           string            `json:"end_time"`
+	MetaData          map[string]string `json:"meta_data"`
+}
```

Comment:
The JSON tags here aren't what uBazaar expects. Should be:
```suggestion
type event struct {
	EventID           string            `json:"eventID"`
	ServiceCustomerID string            `json:"serviceCustomerID"`
	Service           string            `json:"service"`
	UnitOfMeasure     string            `json:"unitOfMeasure"`
	Quantity          float64           `json:"quantity"`
	StartTime         string            `json:"startTime"`
	EndTime           string            `json:"endTime"`
	MetaData          map[string]string `json:"metadata"`
}
```